paper-api/src/main/java/io/papermc/paper/event/inventory/ItemCraftedEvent.java

```diff
@@ -0,0 +1,53 @@
+package io.papermc.paper.event.inventory;
+
+import org.bukkit.entity.Player;
+import org.bukkit.event.Event;
+import org.bukkit.event.HandlerList;
+import org.bukkit.inventory.ItemStack;
+import org.jspecify.annotations.NonNull;
+
+
+/**
+ * Called when a player picks up a crafted item from the result slot of a crafting grid.
+ */
+public class ItemCraftedEvent extends Event{
```

Comment:
You should annotate the class as `@NullMarked` and remove the individual `@NonNull` annotations

Reply:
@orang3i you should add this annotation to the class so you don't need to annotate any field/parameter at all.
```suggestion
@NullMarked
public class ItemCraftedEvent extends Event{
```